locopy/database.py

```diff
@@ -110,7 +110,7 @@ def _disconnect(self):
         else:
             logger.info("No connection to close")
 
-    def execute(self, sql, commit=True, params=None):
+    def execute(self, sql, commit=True, params=()):
```

Comment:
I am curious about this change here. Is this to align with some convention for SQLite?

Reply:
Yeah, when I tried to use `Database` with `params=None` it didn't work